src/main/resources/interface-specification.yml

```diff
@@ -0,0 +1,398 @@
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+paths:
+  '/attributes':
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      summary: Creates attribute
+      tags:
+        - attribute
+      operationId: createAttribute
+      responses:
+        '201':
+          description: Attribute created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Attribute'
+        '400':
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '409':
+          description: Conflict
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+      description: Creates the attribute passed as payload
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AttributeSeed'
+        description: Contains the attribute data to save.
+    get:
+      summary: 'returns the list of attributes available on the registry'
+      tags:
+        - attribute
+      operationId: getAttributes
+      parameters:
+        - in: query
+          name: search
+          schema:
+            type: string
+      responses:
+        '200':
+          description: array of currently available attributes
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AttributesResponse'
+        '404':
+          description: Attributes not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+      description: Returns the list of currently available attributes
+  '/bulk/attributes':
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    get:
+      summary: 'returns the list of attributes for a given bulked ids request'
+      tags:
+        - attribute
+      operationId: getBulkedAttributes
+      responses:
+        '200':
+          description: array of attributes
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AttributesResponse'
+      description: Returns the list of attributes indicated in the request
+      parameters:
+        - in: query
+          name: ids
+          schema:
+            type: string
+          description: 'a string representing the attribute ids concatenated by ","'
+  '/attributes/name/{name}':
```

Comment:
if I'm not wrong, attributes names are no more unique.
Does it make sense to remove this endpoint in favour of an additional parameter in the listing?
cc @beetlecrunch

Reply:
No, they are still unique.